src/users/dto/update-user-dto.ts

```diff
@@ -0,0 +1,6 @@
+export class UpdateUserDto {
+  login: string;
+  password: string;
+  age: number;
+  isDeleted: false;
```

Comment:
I wouldn't go that far to making a statement that it always will be the `false` there...

Reply:
Done.